libraries/AP_Scripting/applets/forward_flight_motor_shutdown.lua

```diff
@@ -39,12 +37,6 @@ if throttle_off_threshold < 100 then
   assert((throttle_off_threshold < throttle_on_threshold) and (throttle_on_threshold < 100), "throttle on and off thresholds not configured correctly")
 end
 
--- clear vars we don't need anymore
-slew_down_time = nil
-slew_up_time = nil
-pwm_range = nil
-
-
```

Comment:
Could enclose the above in a `do ... end` block to create a new scope and actually destroy them but since they are just numbers it is probably a net negative.